code/modules/antagonists/changeling/powers/augmented_eyesight.dm

```diff
@@ -27,12 +27,12 @@
 	var/obj/item/organ/eyes/E = user.getorganslot(ORGAN_SLOT_EYES)
 	if(E)
 		if(!active)
-			E.sight_flags |= SEE_MOBS | SEE_OBJS | SEE_TURFS //Add sight flags to the user's eyes
+			E.sight_flags |= SEE_MOBS //Add sight flags to the user's eyes
```

Comment:
While you are at it, I think you should make this ability use the `TRAIT_THERMAL_VISION` trait instead of eye sight_flags, so it doesn't conflict with stuff like xeno or thermal eyes.